packages/cli/src/Doctor.ts

```diff
@@ -87,13 +86,14 @@ ${chalk.bold('Examples')}
       throw new Error(`${canConnect.code}: ${canConnect.message}`)
     }
 
-    const engine = new IntrospectionEngine({
-      cwd: path.dirname(schemaPath),
+    const engine = new MigrateEngine({
+      projectDir: path.dirname(schemaPath),
```

Comment:
For later: Would `cwd` maybe be a better name for `projectDir` as well?

Reply:
That's a potential choice, yeah